src/map/pet.c

```diff
@@ -592,7 +592,7 @@ static int pet_catch_process2(struct map_session_data *sd, int target_id)
 		return 1;
 	}
 
-	pet_catch_rate = (pet->db[i].capture + (sd->status.base_level - md->level)*30 + sd->battle_status.luk*20)*(200 - get_percentage(md->status.hp, md->status.max_hp))/100;
+	pet_catch_rate = pet->db[i].capture * (100 - get_percentage(md->status.hp, md->status.max_hp)) / 100 + pet->db[i].capture;
```

Comment:
Since this is quite a big reduction of the success rate, I can see how controversial this change might be on an already launched server. What do you think about offering a battle flag to toggle between the custom eAthena formula and the official formula, like we do for various other things (and defaulting to the official formula)?

Reply:
Sounds reasonable. I'll implement this.

Reply:
Done.